src/tmpfiles/tmpfiles.c

```diff
@@ -3861,6 +3867,14 @@ static int parse_line(
                         if (!i.argument)
                                 return log_oom();
                 }
+
+                if (i.ignore_if_missing && access_nofollow(i.argument, F_OK) == -ENOENT) {
```

Comment:
needs to take the chose root fs into account. chase_and_accessat()

Reply:
and I think i'd prefer if we not mix up parsing from application of a symlink, already because tmpfiles.d/ drop-ins might make stuff available via inode creation, hence we should process this test in order.

or in other words, please move this to create_symlink()